dashboard/views/dashboard.ejs

```diff
@@ -54,6 +54,22 @@
 							</div>
 						</div>
 					</div>
+					<div class="col-lg-4 col-md-4 col-sm-4">
```

Comment:
change the column width to `col-lg-3 col-md-3 col-sm-3` so that they appear in a line.

Reply:
do it for all 4 cards